components/atom/input/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@s-ui/react-atom-input",
-  "version": "3.3.0",
```

Comment:
Don't update manually the version as it will be updated for you when releasing the new version of the component.

Reply:
He dejado el package.json como estaba. Thank you